src/tracer.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+import * as Constants from './constants';
+import Span from './span';
+
+/**
+ *
+ */
+export default class Tracer {
+
+    /**
+     * [constructor description]
+     * @return {[type]} [description]
+     */
+    constructor() {
+        this._imp = null;
+    }
+
+    /**
+     * Handle to implementation object.
+     */
+    imp() {
+        return this._imp;
+    }
+
+    /**
+     * TODO: Unclear what the OpenTracing specification wants here.  This
+     * implementation is a best-guess approximation that's likely not right.
+     * The spec refers to text and binary representations, but what does
+     * "binary" mean in JavaScript without making assumptions about protocols,
+     * transports, and platforms?
```

Comment:
Agreed that the spec should be clearer... I'll make an appropriate change in that pending spec PR.

"binary" means an arbitrary-length byte array (as opposed to a utf8 string). "text" means a `string->string` map in whatever data structure is conventional. See also: https://github.com/opentracing/opentracing-java/pull/4/files#diff-1f61a455f742fcd354fedb93c07d28e8R24